web-common/src/components/icons/Expand.svelte

```diff
@@ -0,0 +1,29 @@
+<script>
+  export let size = "1em";
+  export let color = "currentColor";
+</script>
+
+<svg
+  width={size}
+  height={size}
+  viewBox="0 0 16 16"
+  fill={color}
+  xmlns="http://www.w3.org/2000/svg"
+>
+  <path
+    d="M10.2286 2.5C10.2286 2.22386 10.4524 2 10.7286 2H13.5C13.7761 2 14 2.22386 14 2.5V5.27143C14 5.54757 13.7761 5.77143 13.5 5.77143H13.4714C13.1953 5.77143 12.9714 5.54757 12.9714 5.27143V3.75588L10.4107 6.31661C10.2154 6.51188 9.89885 6.51188 9.70359 6.31661L9.68339 6.29641C9.48812 6.10115 9.48812 5.78457 9.68339 5.5893L12.2441 3.02857H10.7286C10.4524 3.02857 10.2286 2.80471 10.2286 2.52857V2.5Z"
+    fill="#374151"
```

Comment:
Should we be hardcoding the fill colour? Perhaps makes sense to match other icons?

Reply:
Most of our icons as of now have hard-coded fill color as they are always used as is. We modify them when we have a use case to show the icon with a different color as and when required.